scripts-dev/build_debian_packages.py

```diff
@@ -23,7 +23,6 @@
 # These are expanded inside the dockerfile to be a fully qualified image name.
 # e.g. docker.io/library/debian:bullseye
 DISTS = (
-    "debian:buster",  # oldstable: EOL 2022-08
```

Comment:
For reference, this fixes failures we're seeing in CI like https://github.com/matrix-org/synapse/actions/runs/5484040336/jobs/9991067272


```
ERROR: Package 'matrix-synapse' requires a different Python: 3.7.13 not in '<4.0.0,>=3.8.0'

...

Error: Command ['pip', 'install', '/tmp/cibuildwheel/repaired_wheel/matrix_synapse-1.87.0-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl'] failed with code 1. 
```

(notice how it's trying to run with an image called `cp37` because that's what Debian Buster has but we dropped support for Python 3.7 already)